Makefile

```diff
@@ -6,16 +6,45 @@ ifneq ($(OS),Windows_NT)
 	OS := $(shell uname)
 endif
 
+.DEFAULT_GOAL := help
+
+.ONESHELL:
+
+.SHELLFLAGS = -ec
+
+VENV = ./venv
```

Comment:
I don't like overriding uv's default of `.venv` here.

Reply:
No problem, I'll undo that.